### ISSUE TITLE
[Question] Unable to click the disabled Element in the webpage

### ISSUE THREAD

**Issue by vidhuvr** [microsoft/playwright#19983]
There is an element which is supposed to be enabled and its aria-disabled attribute is true even though it  is enabled in UI. When I click element, it says the element is not enabled. I used force option to click the element, but I was not able to.

![91276](https://user-images.githubusercontent.com/47219533/211492693-2749afa6-9889-4870-ab4c-fd6e61fd3390.jpg)

**Comment by soumya-kar** [comment]
You can try some thing like this if you are using Python
expect(signup_btn).to_be_enabled(timeout=60000)

This will wait for max up to 60 seconds for element to be enabled.

**Comment by mikalaikarol** [comment]
I just faced exactly the same case as @vidhuvr.
It seems that, in case the element has "aria-disabled" attribute is "true", playwright sees it as disabled and non interactable, even though in UI it is clickable and usable.

`<input type="submit" value="Log in" title="Log in" aria-disabled="true" class="btn btn-primary mb-3 mt-3 btn-lg">`

<img width="557" height="72" alt="Image" src="https://github.com/user-attachments/assets/81f50345-32e5-47b5-9407-9f8cce62e31e" />

I have found a workaround though. 
What helped me is to use .click({ force: true })
